src/conversion.js

```diff
@@ -48,7 +48,7 @@ export function createMetadata (remote: RemoteDoc): Metadata {
 
 // Extract the remote path and name from a local id
 export function extractDirAndName (id: string): [string, string] {
-  const dir = path.dirname(`/${id}`)
+  const dir = '/' + id.split(path.sep).slice(0, -1).join('/')
```

Comment:
I'm surprised that `path.basename` works with windows paths, but not `path.dirname`. Maybe it is just the leading `/`